Query was executed successfully

[An error expected at this place for the query "load(A,'../tests/harness/testcases/data/negative_idx.txt')". And it failed with error code = scidb::SCIDB_SE_IMPORT_ERROR::SCIDB_LE_FILE_IMPORT_FAILED. Expected error code = scidb::SCIDB_SE_IMPORT_ERROR::SCIDB_LE_FILE_IMPORT_FAILED.]

Query was executed successfully

